test/Dialect/FIRRTL/specialize-layers.mlir

```diff
@@ -29,6 +29,15 @@ firrtl.circuit "DisableTwice" attributes {
   firrtl.extmodule @DisableTwice()
 }
 
+// CHECK-LABEL: firrtl.circuit "DefaultSpecialization"
+firrtl.circuit "DefaultSpecialization" attributes {
+    // This option does not get removed - it is idempotent to leave it.
+    // CHECK: default_layer_specialization = #firrtl<layerspecialization enable>
+    default_layer_specialization = #firrtl<layerspecialization enable>
+  } {
+  firrtl.extmodule @DefaultSpecialization()
+}
```

Comment:
I agree with this. It originally seemed incongruous to remove the attributes for enabling/disabling, but leave this one. However, this one does make sense to leave in.